DigitalLearningSolutions.Web/Views/Login/ChooseACentre.cshtml

```diff
@@ -0,0 +1,60 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Login
+@model List<ChooseACentreViewModel>
+@{
+  ViewData["Title"] = "Choose A Centre";
+}
+
+<link rel="stylesheet" href="@Url.Content("~/css/login/chooseACentre.css")" asp-append-version="true">
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+
+    <h1 id="page-heading" class="nhsuk-heading-xl">Choose a Centre</h1>
+    <p class="nhsuk-body-l">
+      You are registered at more than one centre. Please choose the centre you would like to sign in to.
+    </p>
+
+    <table role="table" class="nhsuk-table-responsive">
+      <thead role="rowgroup" class="nhsuk-table__head">
+      <tr role="row">
+        <th role="columnheader" class="" scope="col">
+          Centre
+        </th>
+        <th role="columnheader" class="" scope="col">
+          Roles
+        </th>
+        <th role="columnheader" class="" scope="col">
+          Action
+        </th>
+      </tr>
+      </thead>
+      <tbody class="nhsuk-table__body">
+      @foreach (var centreRow in Model) {
+        <form method="post" novalidate asp-action="ChooseACentre">
+          <tr role="row" class="nhsuk-table__row">
+            <td role="cell" class="nhsuk-table__cell">
+              @centreRow.CentreName
+            </td>
+            <td role="cell" class="nhsuk-table__cell">
+              @if (centreRow.IsAdmin) {
+                <strong class="nhsuk-tag nhsuk-tag--blue centre-role-tag">Admin</strong>
+              }
+              @if (centreRow.IsDelegate) {
+                <strong class="nhsuk-tag nhsuk-tag--grey centre-role-tag">Delegate</strong>
+              }
+            </td>
+            <td role="cell"class="nhsuk-table__cell">
+              <input type="hidden" name="centreId" value="@centreRow.CentreId"/>
+              <button class="nhsuk-button nhsuk-u-margin-bottom-1 nhsuk-u-padding-bottom-0 nhsuk-u-padding-top-0" type="submit">
```

Comment:
I wonder if instead of making this a form and adding this hidden input, we could navigate to a url where centre id is either embedded in the url or a query string (e.g. /Centre/1/Login or /Login?CentreId=1 or /ChooseCentre?CentreId=1) 🤔 Then this can be a simple anchor tag with a role button rather than a form submission, which is a bit odd after the user has already submitted a form (not 100% sure how this will work with a screenreader)